src/Sentry/Infrastructure/ConsoleDiagnosticLogger.cs

```diff
@@ -9,25 +8,16 @@ namespace Sentry.Infrastructure
     /// <remarks>
     /// The default logger, usually replaced by a higher level logging adapter like Microsoft.Extensions.Logging.
     /// </remarks>
-    public class ConsoleDiagnosticLogger : IDiagnosticLogger
+    public class ConsoleDiagnosticLogger : DiagnosticLogger
     {
-        private readonly SentryLevel _minimalLevel;
-
         /// <summary>
         /// Creates a new instance of <see cref="ConsoleDiagnosticLogger"/>.
         /// </summary>
-        public ConsoleDiagnosticLogger(SentryLevel minimalLevel) => _minimalLevel = minimalLevel;
-
-        /// <summary>
-        /// Whether the logger is enabled to the defined level.
-        /// </summary>
-        public bool IsEnabled(SentryLevel level) => level >= _minimalLevel;
```

Comment:
was this not used? and isnt this a breaking change?

Reply:
ahh i see. base class. ignore me

Reply:
np. :)